frontend/src/components/videoPlayer/Comments.js

```diff
@@ -28,79 +27,43 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Comments = () => {
+const Comments = ({ movie }) => {
   const classes = useStyles();
```

Comment:
Since you only need the comments from the movie object you could send just the comments as a prop.